src/controllers/block.py

```diff
@@ -52,7 +52,8 @@ def block_attribute(self, attribute):
         if attribute not in self.block_columns:
             return("Not valid attribute")
         return_attributes = {}
-        for key in self.to_json().keys():
-            if key != attribute and key != "_id":
-                return_attributes[key] = self.to_json()[key]
+        # for key in self.to_json().keys():
```

Comment:
Remove commented code (thats one of the benefits of using Git, you can delete code, an still look it up in the previous version if you need it)